docs/application/dotnet/guides/nui/window.md

```diff
@@ -0,0 +1,289 @@
+{
+    if (e.Touch.GetState(0) == PointStateType.Down)
+    {
+        // The window has been touched, do something
+    }
+}
+```
+
+### Key event
+
+Key event is emitted when the window receives a key event from the window manager.
+
+The following code shows how to handle the key event:
+
+```csharp
+window.KeyEvent += OnWindowKeyEvent;
+
+private void OnWindowKeyEvent(object sender, Window.KeyEventArgs e)
+{
+    if (e.Key.State == Key.StateType.Down)
+    {
+        if (e.Key.KeyPressedName == "Left")
+        {
+            // The left arrow key is pressed, do something
+        }
+        else if (e.Key.KeyPressedName == "Right")
+        {
+            // The right arrow key is pressed, do something
+        }
+    }
+}
+```
+
+### Resize event
+
+Resize event is emitted when the window is resized.
+
+The following code shows how to handle the resize event:
+
+```csharp
+window.Resized += OnWindowResized;
+
+private void OnWindowResized(object sender, Window.ResizedEventArgs e)
+{
+    // Window is resized, do something
+}
+```
+
+For more information, see [Window](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html) API.
+
+## Window rotation
+
+To support rotation, NUI provides two methods.
+// To do: tscholb
+
+### Enable or disable window
+
+If the device supports rotation, application can rotate the window with specific orientation.
+
+The supported orientation are `Portrait`, `Landscape`, `PortraitInverse` and `LandscapeInverse`.
+`Portrait` means the height is larger than the width or same.
+`Landscape` means the width is larger than the height.
+
+<img src="./media/device_rotate2.png">
+
+
+Application can enable or disable a specific orientation to use orientation functions. The following code shows that `Portrait`, `Landscape`, `PortraitInverse` and `LandscapeInverse` are supported in the application.
+If the device's width is larger than the height, then the window will be shown as landscape when application is launched. If the device's width is smaller than the height or same, then the window will be shown as portrait.
+```csharp
+// To enable Portrait
+window.AddAvailableOrientation( Portrait );
+
+// To enable Landscape
+window.AddAvailableOrientation( Landscape );
+
+// To disable PortraitInverse
+window.RemoveAvailableOrientation( PortraitInverse );
+
+// To enable LandscapeInverse
+window.AddAvailableOrientation( LandscapeInverse );
+```
+
+
+### Preferred orientation
+
+If you want the window to appear in a specific orientation, the preferred orientation function is supported. The preferred orientation is one of the available orientations. For example, as shown in the following code, the available orientation list has `Portrait`, `Landscape`, and `LandscapeInverse`, and the preferred orientation is set to Landscape. This implies that the window will appear in landscape even when the device is rotated with any other orientation.
```

Comment:
Please add colon instead of a period at the end of the sentence.

```suggestion
If you want the window to appear in a specific orientation, the preferred orientation function is supported. The preferred orientation is one of the available orientations. For example, as shown in the following code, the available orientation list has `Portrait`, `Landscape`, and `LandscapeInverse`, and the preferred orientation is set to Landscape. This implies that the window will appear in landscape even when the device is rotated with any other orientation:
```